### PR TITLE
Add Logs to HttpRegistryManager

### PR DESCRIPTION
**Pull request by azabbasi**

### CODE REVIEW

File: iothub/service/src/HttpRegistryManager.cs [hunk 1]
```diff
@@ -31,8 +31,6 @@ internal class HttpRegistryManager : RegistryManager
         private const string PageSizeHeader = "x-ms-max-item-count";
 
         private const string TwinUriFormat = "/twins/{0}?{1}";
-        private const string TwinTagsUriFormat = "/twins/{0}/tags?{1}";
```
Comment (azabbasi) [inline iothub/service/src/HttpRegistryManager.cs:34]: Not used

File: iothub/service/src/HttpRegistryManager.cs [hunk 2]
```diff
@@ -497,17 +681,33 @@ private Task RemoveConfigurationAsync(string configurationId, IETagHolder eTagHo
             return _httpClientHelper.DeleteAsync(GetConfigurationRequestUri(configurationId), eTagHolder, errorMappingOverrides, null, cancellationToken);
         }
 
+        [Obsolete]
```
Comment (abhipsaMisra) [inline iothub/service/src/HttpRegistryManager.cs:684]: really?

File: iothub/service/src/HttpRegistryManager.cs [hunk 3]
```diff
@@ -1512,18 +2066,6 @@ private static Uri GetTwinUri(string deviceId)
             return new Uri(TwinUriFormat.FormatInvariant(deviceId, ClientApiVersionHelper.ApiVersionQueryString), UriKind.Relative);
         }
 
-        private static Uri GetTwinTagsUri(string deviceId)
```
Comment (abhipsaMisra) [inline iothub/service/src/HttpRegistryManager.cs:1515]: unused?

**Review by abhipsaMisra — APPROVED** [review]
My only comment was regarding the APIs being marked as Obsolete, which Azad alerady explained to me offline.
Looks good!

File: iothub/service/src/HttpRegistryManager.cs (hunk 3 above)
Comment (azabbasi) [reply iothub/service/src/HttpRegistryManager.cs:1515]: yes  yes

File: iothub/service/src/HttpRegistryManager.cs (hunk 2 above)
Comment (azabbasi) [reply iothub/service/src/HttpRegistryManager.cs]: yes yes

**Comment by azabbasi** [comment]
azp run

**Pull request merged**